### DISCUSSION TITLE
Question about using a non-Let's Encrypt provider

### DISCUSSION THREAD

**Discussion by joecoolish** [shibayan/appservice-acmebot#306] [Q&A] [answered]
I am trying to figure out how to switch endpoint providers.  I changed the App Setting to point to `https://api.buypass.com/acme/` but the service is still trying to issue certs from Let's Encrypt:

![image](https://user-images.githubusercontent.com/1739416/112505475-324b3380-8d63-11eb-9c6a-bcad835f9de6.png)

And here is the Application Map from App Insights showing that letsencrypt.org is still being called:

![image](https://user-images.githubusercontent.com/1739416/112505704-6a527680-8d63-11eb-8f7b-c8f0fe233230.png)

How do I fully change over to another provider?

**Comment by shibayan** [comment, marked as answer]
Once Acmebot is running, the Endpoint settings will be cached in `D:\home\.acme`. If you want to use a different endpoint, you will need to rename or delete this directory.
